extensions/capture/README.md

```diff
@@ -0,0 +1,3 @@
+# capture
+
+Quickly capture to your Capture Inbox with Raycast.
```

Comment:
This reads a bit weird to me, but again I'm not a native english speaker 😅 I guess this relates to your Capture nomenclature. After I watched your video I was wondering if it wouldn't make more sense to call it "Create records in your Capture Inbox with Raycast" ?

Reply:
Or at least add "new record" to the sentence. "Quickly capture new record to..."